Neos.ContentRepositoryRegistry/Classes/Factory/NodeTypeManager/DefaultNodeTypeManagerFactory.php

```diff
@@ -24,8 +24,7 @@ function () {
                 $configuration = $this->configurationManager->getConfiguration('NodeTypes');
                 return $this->nodeTypeEnrichmentService->enrichNodeTypeLabelsConfiguration($configuration);
             },
-            $this->nodeLabelGeneratorFactory,
-            $options['fallbackNodeTypeName'] ?? null,
```

Comment:
i guess we have to also remove the yaml config for it?

Reply:
done